Packs/FireEyeETP/Integrations/FireEyeETP/FireEyeETP.py

```diff
@@ -478,6 +558,86 @@ def get_alerts_command():
     demisto.results(entry)
 
 
+def upload_yara_file_command(client, args):
+
+    entry_id = args.get('entryID')
+    policy_uuid = args.get('policy_uuid')
+    ruleset_uuid = args.get('ruleset_uuid')
+
+    file_obj = demisto.getFilePath(entry_id)
+    file_path = file_obj['path']
+
+    with open(file_obj.get('path'), "rb") as file:
+        data=file.read()
+        files = {
+            'file': ('new.yara', data)
+        }
+        response = client.upload_yara_file(policy_uuid, ruleset_uuid, files)
+        #print (response2)
+        if response.status_code == 202:
+            return "Upload of Yara file succesfully."
+        else :
+            return "Upload of Yara file failed."
+
+
+def get_events_data_command(client, args):
+
+    message_id = args.get('message_id')
+
+    response = client.get_events_data(message_id)
+
+
+    result_output = dict()
+    result_output['Logs'] = response['data'][message_id]
+
+    for log in result_output['Logs']:
+        if log['action_on_msg'] == "MTA_RCPT_DELIVERED_OUTBOUND":
+            result_output['Delivered_msg'] = log['display_msg']
+            result_output['Delivered_status'] = "Delivered"
+            result_output['InternetMessageId'] = result_output['Delivered_msg'].split('<')[1].split('>')[0]
+        if log['action_on_msg'] == "MTA_RCPT_DELIVERY_PERM_FAILURE_OUTBOUND":
+            result_output['Delivered_msg'] = log['display_msg']
+            result_output['Delivered_status'] = "Failed"
+
+    command_results = CommandResults(outputs=result_output, outputs_prefix='FireEyeETP.Events')
+    return command_results
+
+
+def download_alert_artifacts_command(client,args):
+    alert_id = args.get('alert_id')
+
+    response = client.get_artifacts(alert_id)
+    file_entry = fileResult(alert_id + '.zip', data=response.content, file_type=EntryType.FILE)
+
+    return file_entry
```

Comment:
Please return also CommandResults which indicates the command was successfully finished. 
Something like: return [CommandResults(readable_output='Download alert artifact completed successfully'), file_entry]
it should output both